### PR TITLE
chore(mysql): migrate mysql examples

### PR DESCRIPTION
**Pull request by haddasbronfman**
## Which problem is this PR solving?

Migrate examples for mysql instrumentation: https://github.com/open-telemetry/opentelemetry-js-contrib/issues/1163

## Short description of the changes

The mysql examples were moved from ./examples/mysql to ./plugins/node/opentelemetry-instrumentation-mysql/examples

## Checklist

- [ ] Ran `npm run test-all-versions` for the edited package(s) on the latest commit if applicable.

### CODE REVIEW

**Comment by dyladan** [comment]
Looks like you didn't remove the original files in ./examples/mysql

**Comment by haddasbronfman** [comment]
@dyladan You're right. this is because I read here: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/939 that: _"It was discussed that we could keep the top-level examples directory for 2 reasons"_
Together with that, I saw the 'express' examples are removed, and this is why I remove also 'koa' examples when I handled the koa migration. Anyway - I'm confused now and don't sure which approach should I go. keep or delete? 
thanks.

**Comment by dyladan** [comment]
We are keeping it for examples not yet migrated and for examples which show basic functionality that doesn't apply to a specific instrumentation. The same example should not be maintained in two places so the original files should be deleted

**Comment by haddasbronfman** [comment]
@dyladan fixed. thanks for the clarification.

**Comment by dyladan** [comment]
Sorry i didn't get around to proper reviews of this today with SIG and such but it's on my list for the morning

**Review by blumamir — APPROVED** [review]

File: plugins/node/opentelemetry-instrumentation-mysql/examples/src/tracer.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+import opentelemetry from '@opentelemetry/api';
```
[the remark this answers is not in the document]
Comment (dyladan) [reply plugins/node/opentelemetry-instrumentation-mysql/examples/src/tracer.ts:3]: no the api has a default export. doing it this way is fine

File: plugins/node/opentelemetry-instrumentation-mysql/examples/src/tracer.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+export const setupTracing = (serviceName: string) => {
+  const provider = new NodeTracerProvider({
+    resource: new Resource({
+    [SemanticResourceAttributes.SERVICE_NAME]: serviceName,
+  }),});
+
+  provider.addSpanProcessor(new SimpleSpanProcessor(new ZipkinExporter()));
+  provider.addSpanProcessor(new SimpleSpanProcessor(new JaegerExporter()));
```
[the remark this answers is not in the document]
Comment (dyladan) [reply plugins/node/opentelemetry-instrumentation-mysql/examples/src/tracer.ts:22]: yeah i agree here

**Review by dyladan — APPROVED** [review]
LGTM thanks for doing this

File: plugins/node/opentelemetry-instrumentation-mysql/examples/package.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    "@opentelemetry/instrumentation": "^0.32.0",
+    "@opentelemetry/instrumentation-http": "^0.32.0",
+    "@opentelemetry/instrumentation-mysql": "^0.31.0",
+    "@opentelemetry/sdk-trace-base": "^1.0.0",
+    "@opentelemetry/sdk-trace-node": "^1.0.0",
+    "mysql": "^2.18.1"
+  },
+  "homepage": "https://github.com/open-telemetry/opentelemetry-js#readme",
+  "devDependencies": {
+    "@types/mysql": "*",
```
[the remark this answers is not in the document]
Comment (haddasbronfman) [reply plugins/node/opentelemetry-instrumentation-mysql/examples/package.json:46]: fix to ^2.15.21

File: plugins/node/opentelemetry-instrumentation-mysql/examples/src/tracer.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (haddasbronfman) [reply plugins/node/opentelemetry-instrumentation-mysql/examples/src/tracer.ts:22]: fixed

**Review by vmarchaud — APPROVED** [review]

**Pull request merged**